tests/actions/draftActions.test.js

```diff
@@ -131,14 +132,33 @@ describe("Draft Actions", () => {
     done();
   });
 
-  it("should dispatch DELETE_ALL_ITEMS_IN_DRAFT_LAB_ORDER", async done => {
+  it("should dispatch DELETE_DRAFT_DRUG_ORDER_SUCCESS", async done => {
+    const order = {
+      drugName: "paracetamol",
+      drug: "23sdfert4356reg4321"
+    };
+
+    const draftType = "drugorder";
     const expectedActions = {
-      type: DELETE_ALL_ITEMS_IN_DRAFT_LAB_ORDER
+      type: DELETE_DRAFT_DRUG_ORDER_SUCCESS,
+      order: order
     };
 
     const store = mockStore({});
-    await store.dispatch(discardTestsInDraft());
+    await store.dispatch(discardTestsInDraft({ order, draftType }));
     expect(store.getActions()).toEqual([expectedActions]);
     done();
   });
+
+  it("should dispatch both and DELETE_ALL_ITEMS_IN_DRAFT_LAB_ORDER", async done => {
+    const expectedActions = [
```

Comment:
Some texts are missing in this test description:
```
it("should dispatch both and DELETE_ALL_ITEMS_IN_DRAFT_LAB_ORDER"....
```

Reply:
Thanks for pointing this out. I have made the appropriate modification.